.github/workflows/Sweave2rmd.yaml

```diff
@@ -16,34 +17,45 @@ jobs:
       R_KEEP_PKG_SOURCE: yes
       REPO_NAME: ${{ github.event.repository.name }}
     steps:
+
       - uses: actions/checkout@v3
-      - name: create repo
-        run: |
-          echo $REPO_NAME
-          git clone "https://git.bioconductor.org/packages/$REPO_NAME"
-          rm -rf $REPO_NAME/.git*
-          mv $REPO_NAME/* .
-          rm -rf $REPO_NAME
```

Comment:
Add back

```
      - name: create repo
        run: |
          echo $REPO_NAME
          git clone "https://git.bioconductor.org/packages/$REPO_NAME"
          rm -rf $REPO_NAME/.git*
          mv $REPO_NAME/* .
          rm -rf $REPO_NAME
```

In the future, this should probably be replaced with a workflow_dispatch input to specify the repository to clone.